sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -980,33 +980,36 @@ object SQLConf {
       .createWithDefault(true)
 
   val FILES_MAX_PARTITION_BYTES = buildConf("spark.sql.files.maxPartitionBytes")
-    .doc("The maximum number of bytes to pack into a single partition when reading files.")
+    .doc("The maximum number of bytes to pack into a single partition when Spark file-based " +
+      "sources are used to read files.")
```

Comment:
Can you just revert everything here and add a sentence like "This configuration is effective only when using file-based sources such as Parquet, JSON and ORC." at the end of each configuration?

Reply:
make sense, thanks.